_layouts/exit.html

```diff
@@ -0,0 +1,8 @@
+{% include header.html %}
+<main id="main-content" class="grid-container">
+    {{ content }}
```

Comment:
This is looking good in terms of functionality. For who (q1), we have to get the "other resources" content back in there. It could either be done with an "if" statement (see line 15 of header.html in _includes), or you could move that HTML into the .md file.